package.json

```diff
@@ -37,7 +37,7 @@
     "posttest": "agadoo internal/index.mjs",
     "prepublishOnly": "npm run lint && PUBLISH=true npm test",
     "tsd": "tsc -p src/compiler --emitDeclarationOnly && tsc -p src/runtime --emitDeclarationOnly",
-    "lint": "eslint '{src,test}/**/*.{ts,js}'"
+    "lint": "eslint \"{src,test}/**/*.{ts,js}\""
```

Comment:
I'm not sure I would have changed this line

Reply:
It wasn't really related to the sourcemap/preprocessor test changes, but I think it is necessary for linting to work on Windows.